daemons/fenced/fenced_commands.c

```diff
@@ -2580,16 +2580,78 @@ cancel_stonith_command(async_command_t * cmd)
     }
 }
 
+/*!
+ * \internal
+ * \brief Cancel and reply to any duplicates of a just-completed operation
+ *
+ * Check whether any fencing operations are scheduled to do the same thing that
+ * as one that just succeeded. If so, rather than performing the same operation
```

Comment:
should be fine without 'that' in the above line?

Reply:
The one after 'thing'

Reply:
whoops, yes